package.json

```diff
@@ -5,12 +5,11 @@
   "description": "AWS SDK for JavaScript from the future",
   "main": "index.js",
   "scripts": {
+    "generate-clients": "node ./scripts/generate-clients",
     "bootstrap": "yarn",
     "clean": "yarn clear-build-cache && yarn clear-build-info && lerna clean",
     "clear-build-cache": "rimraf ./packages/*/build/* ./clients/*/*/build/*",
     "clear-build-info": "rimraf ./packages/*/*.tsbuildinfo ./clients/*/*/*.tsbuildinfo",
-    "copy-models": "node ./scripts/copyModels.js",
-    "update-clients": "node ./packages/package-generator/build/cli.js import-all --matching './models/*/*/service-2.json'",
     "build:crypto-dependencies": "lerna run --scope '@aws-sdk/types' --scope '@aws-sdk/util-utf8-browser' --scope '@aws-sdk/util-locate-window' --scope '@aws-sdk/hash-node' --include-filtered-dependencies pretest",
     "build:smithy-client": "lerna run --scope '@aws-sdk/client-rds-data' --include-filtered-dependencies pretest",
```

Comment:
Can build:smithy-client get dropped now?

Reply:
This is still needed if we want to test the build of our only client -- `client-rds-data`

Reply:
Ok.  We may want to cut-over to using the new script so it gets exercised.  Codegen for a single model should be pretty quick.

Reply:
`build:smithy-client` is for compiling the client and its dependencies from TS to JS. It basically has nothing todo with the codegen script